gpkit/nomials/map.py

```diff
@@ -20,8 +20,9 @@ class NomialMap(HashVector):
     x and y are VarKey objects.
     """
     units = None
-    expmap = None  # used for monomial-mapping postsubstitution; see .mmap()
-    csmap = None   # used for monomial-mapping postsubstitution; see .mmap()
+    expmap = None   # used for monomial-mapping postsubstitution; see .mmap()
+    csmap = None    # used for monomial-mapping postsubstitution; see .mmap()
+    varexps = False # used for signomial exponent operations
```

Comment:
convention for comments in GPkit is double space between code and `#`, no alignment

Reply:
I think pylint may complain about one of those or the other?